frontend/src/App.js

```diff
@@ -32,110 +35,21 @@ const Scene = () => {
   const [playing, setPlaying] = useState(false);
   const [soundOn, setSoundOn] = useState(true);
   const [toneArmFinished, setToneArmFinished] = useState(false);
+  const coverPicUrls = ["Aiguille.jpg", "CanaryForest.jpg", "Sworn.jpg"];
```

Comment:
Maybe move this to outside of the functional component and make this variable all CAPS to indicate it's not going to change